posthog/models/channel_type/sql.py

```diff
@@ -0,0 +1,46 @@
+import json
+import os
+
+from posthog.settings import CLICKHOUSE_CLUSTER
+
+GA4_CHANNEL_DEFINITION_TABLE_SQL = f"""
```

Comment:
Same question as before. Does it make sense to call this `GA4`? 2 years from now when it's killed by Google, we'll still be stuck with this number in our codebase.

Reply:
Done